packages/domain-picker/test/search-and-select-domain-suggestion.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { DataStatus, DomainSuggestion } from '@automattic/data-stores/src/domain-suggestions';
+
+/**
+ * Internal dependencies
+ */
+import '../matchMedia.mock';
```

Comment:
Why do we need to mock `window.matchMedia`?

Reply:
Because of the `useViewportMatch` hook used by the domain suggestion item — without mocking `window.matchMedia`, the test would fail

https://github.com/Automattic/wp-calypso/blob/4edfbd6e9347ea3beeb77f5e1273caebc2c3d2bf/packages/domain-picker/src/domain-picker/suggestion-item.tsx#L72